command/var_lock.go

```diff
@@ -73,7 +73,15 @@ Var lock Options:
 	unavailability without causing leader elections, but increases the amount of
 	time required to detect a lost lock in some cases. Defaults to 5. Set to 0 to
 	disable.
-
+  
+  -early-return 
+	Optional, early-return indicates the command to return if the lock is not 
+	acquired instead of waiting on stand by to try again. Defaults to false.
+
+  -backoff
+	Optional, by default the lease algorithm waits for 1.1 times the lock TTL, 
+	using the backoff flag, this backoff period can be modified.
```

Comment:
This isn't clear what the backoff is waiting _for_; the docstring you have in `api` package is more clear. I'd go with something like:

```suggestion
  -backoff
	Optional, indicates how long to wait between attempts to obtain the lock. 
	By default the lease algorithm waits for 1.1 times the lock TTL.
```